core/src/apps/tezos/sign_tx.py

```diff
@@ -71,12 +71,16 @@ async def sign_tx(msg: TezosSignTx, keychain: Keychain) -> TezosSignedTx:
             # operation to transfer tokens from a smart contract to an implicit account or a smart contract
             elif transfer is not None:
                 to = _get_address_from_contract(transfer.destination)
-                await layout.require_confirm_tx(to, transfer.amount)
+                await layout.require_confirm_tx(
+                    to, transfer.amount, chunkify=bool(msg.chunkify)
```

Comment:
small nit: we could get rid of the `bool()` noise by declaring the field `[default=false]` in the protobuf

Reply:
Right, I kind of did not want to do this, as that would make the protobuf definition line longer and all the comments would need to be aligned - causing quite a big protobuf diff.

Poor reason, what do you think about it?

Reply:
let's keep as-is now, and maybe create a separate issue for it?
we want to move the protobuf comment to the line before anyway for #690

Reply:
Agreed, issue created - https://github.com/trezor/trezor-firmware/issues/3274